_posts/2017-05-11-integrating-with-billing.md

```diff
@@ -267,6 +271,33 @@ Sample Response:
 "subscriptionId":"65f713f3-55eb-40ed-bb4a-e4f392ccc2fb","status":"ACTIVE"}
 ~~~
 
-#### (4) Next steps
+#### (4) Get Plan Information
+
+All Plan details for a particular app(using packageName) can be obtained as follows:
+
+For eg:
+~~~
+Request URL - {endpoint}/apps/{packageName}/plans
+
+curl -X GET \
+  https://billing.poynt.net/apps/com.satyavedule.paymentfragment/plans \
```

Comment:
do you want to use the appId to be consistent with pervious example?
`urn:aid:b326335b-ce7c-4482-80d4-109e0fe6f9d9`